packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart

```diff
@@ -356,4 +356,40 @@ abstract class GoogleMapsFlutterPlatform extends PlatformInterface {
   }) {
     throw UnimplementedError('buildView() has not been implemented.');
   }
+
+  /// Returns a widget displaying the map view.
+  ///
+  /// This method is similar to [buildView], but contains a parameter for
+  /// platforms that require a text direction.
+  ///
+  /// Default behavior passes all parameters except `textDirection` to
+  /// [buildView]. This is for backward compatibility with existing
+  /// implementations. Platforms that use the text direction should override
+  /// this as the primary implementation, and delegate to it from buildView.
+  Widget buildViewWithTextDirection(
+    int creationId,
+    PlatformViewCreatedCallback onPlatformViewCreated, {
+    required CameraPosition initialCameraPosition,
+    TextDirection textDirection = TextDirection.ltr,
```

Comment:
Optional: Should this be required, so that callers have to think about it (and set a default themselves if they don't have a good value) instead of just not noticing?

Reply:
I think making it required makes more sense. I changed it.